util/io/src/lib.rs

```diff
@@ -27,33 +27,33 @@
 //! struct MyHandler;
 //!
 //! struct MyMessage {
-//! 	data: u32
+//!    data: u32
```

Comment:
@somniumism Does the rust-fmt change this code?

Reply:
@majecty I changed this code because of clippy warning: `using tabs in doc comments is not recommended` I'm sorry I couldn't explain it in advance.

Reply:
You can split the commit into "Update clippy version" and "Fix clippy error". Since the change is small you can just add a comment on the PR page about why you changed it.